index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="icon" href="Tesla.svg">
+        <link rel="stylesheet" href="style.css" />
+        <title>Elon Musk Tribute</title>
+    </head>
+    
+    <body class="main-container">
+        <div id="main">
+            <h1 id="title" class="heading">Elon Musk</h1>
+            <div class="img-div" id="img-div">
+                <p id="img-caption" class="description">Below is an image of Elon Musk in front of a Tesla Sign</p>
+        <img id="image" class="image" src="musk-tesla.jpg" alt="Picture of Elon Musk"
+          width="800px"/>
```

Comment:
Please make sure you add the true value of `width`, make sure you respect the property of `width` in your `html` code. that is why you have `linter error` in this part.
You can read the following article to know how to attribute the value of `width` in HTML:

- https://www.w3schools.com/tags/att_width.asp